packages/commonwealth/client/scripts/hooks/useAppStatus.ts

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+const useAppStatus = () => {
+  const [isAddedToHomeScreen, setIsAddedToHomeScreen] = React.useState(true);
+  const [isStandalone, setIsStandalone] = React.useState(true);
+  const [isMarketingPage, setIsMarketingPage] = React.useState(false);
+  const [isIOS, setIsIOS] = React.useState(false);
+  const [isAndroid, setIsAndroid] = React.useState(false);
+
+  React.useEffect(() => {
+    setIsStandalone(
+      process.env.NEXT_PUBLIC_ONLY_STANDALONE === 'true' ? true : false,
```

Comment:
1. what this env does, it is documented somewhere?
2. we do not have NEXT so it wont work anyways. We use webpack for custom envs

Reply:
Yup, this was something borrowed from baka, was going to smoothen this before I undrafted the PR